Extension/lib/pages/options.js

```diff
@@ -465,6 +467,30 @@ PageController.prototype = {
         contentPage.sendMessage({type: 'clearWhiteListFilter'});
     },
 
+    _checkSafariContentBlockerRulesLimit: function (rulesCount) {

```

Comment:
All new methods must be documented.


```
Checks Safari content blocker rules limit. It's important to check that limit because of Safari limitations. Content blocker with too many rules won't work at all.
```